docs/_docs/themes.md

```diff
@@ -3,7 +3,7 @@ title: Themes
 permalink: /docs/themes/
 ---
 
-Jekyll has an extensive theme system that allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package up layouts, includes, and stylesheets in a way that can be overridden by your site's content.
+Jekyll has an extensive theme system that allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package up plugins, assets, layouts, includes, and stylesheets in a way that can be overridden by your site's content.
```

Comment:
Theme-gems do not package plugins. The gemspec only *specifies* gem-based plugins that are then `required` by Jekyll while loading the theme-gem.
`*.rb` files in a `_plugins` directory in the theme-gem will not be loaded whatsoever (as of `v3.7.x`)